conductor/__init__.py

```diff
@@ -1,10 +1,14 @@
+import logging
 from conductor.lib import common, loggeria
 
 #The version string is updated by the build system.
 #Do not modify the following line.
 #__version__="0.0.0"
 
 # Read the config yaml file upon module import
+
+logging.basicConfig()
```

Comment:
As commented in the next commit, this was added to suppress a "no handlers found" error when Clarisse starts. 
https://stackoverflow.com/a/44188369/179412
From what I can tell, it doesn't change the behavior of other submitters. It's one of the few places I've changed code in a file that affects submitters other than houdini and clarisse.